tests/integration/test_ec2.py

```diff
@@ -13,23 +13,52 @@
 from awscli.testutils import unittest, aws
 
 
-class BaseEC2Test(unittest.TestCase):
-    def assert_dry_run_success(self, command):
+class TestDescribeInstances(unittest.TestCase):
+    def setUp(self):
+        self.prefix = 'ec2 describe-instances --region us-west-2'
+
+    def test_describe_instances_with_id(self):
+        command = self.prefix + ' --instance-ids malformed-id'
         result = aws(command)
-        expected_response = ('Request would have succeeded, '
-                             'but DryRun flag is set.')
-        self.assertIn(expected_response, result.stderr)
+        self.assertIn('InvalidInstanceID.Malformed', result.stderr)
```

Comment:
I wonder if it is possible if there are any other filter parameter combinations (other than instance-id) that can recreate the regression reliably. Otherwise I am about a zero on adding this test as there is not much value the test adds as even if we reintroduced the regression it would not catch it.

I doubt there will be a different filter parameter combination but it would be good to take a quick look.